### ISSUE TITLE
Bug: No docker manifest for LND 0.10.1-beta

### ISSUE THREAD

**Issue by AnthonyRonning** [jamaljsr/polar#350]
**Describe the bug**
Refreshed the docker images in Polar and saw 10.1 was available, added it to my network and clicked start but after a few short seconds, it came up with a "no manifest" error on that node. Luckily the instructions here (https://github.com/jamaljsr/polar/tree/master/docker#lnd) worked and I was able to manually build the image on my local machine but I believe the problem is that it's not located here: https://hub.docker.com/r/polarlightning/lnd/tags

**To Reproduce**
1. Refresh images in Polar
2. Add 0.10.1-beta to a network (new or old)
3. Click start

**Expected behavior**
Should download and start the 0.10.1-beta container.

**Desktop (please complete the following information):**
- OS: OSX Catalina 
- Polar Version: 1.0.0
- Docker Version: 19.03.8
- Docker Compose Version: 1.25.4

Thanks for mentioning that fallback though, may start doing that for RC builds going forward.

**Comment by jamaljsr** [comment]
Hey @AnthonyRonning, thanks for reporting this issue. You are correct. It appears that I did forget to push the image to docker hub after building and testing it locally. I just pushed it so it shouldn't be an issue for anyone else going forward. I'm glad you were able to work around the issue locally.

I am planning on automating the update process for new node versions. I just need to write more integration tests to ensure that there aren't any compatibility issues caused by the updates. Once this is done, I won't have to worry about forgetting to push the image.

FYI: I am also going to begin including RC builds in the next LND & bitcoind release cycles. This was requested by someone else as well.
